site/pages/docs/ref/plugins-fr.hbs

```diff
@@ -32,7 +32,7 @@
 	<li><a href="menu/menu-fr.html">Menu</a> - Une barre de menu interactive comprenant des sous-menus optionnels.</li>
 	<li><a href="texthighlight/texthighlight-fr.html">Mise en surbrillance de texte</a> - Met en surbrillance sur une page Web les mots choisis.</li>
 	<li><a href="footnotes/footnotes-fr.html">Notes de bas de page</a> - Une méthode cohérente et facile d’accès pour répertorier les notes de bas de page.</li>
-	<li><a href="prettify/prettify-fr.html">Prettify</a> - Affiche de la syntaxe des extraits de code source.</li>
+    <li><a href="prettify/prettify-fr.html">Prettify</a> - Coloration syntaxique des extraits de code source dans une page html en utilisant <a href="https://github.com/google/code-prettify" rel="external" hreflang="en">google-code-prettify</a>.</li>
```

Comment:
https://marketplace.visualstudio.com/items?itemName=EditorConfigTeam.EditorConfig will help you.